WeakAuras/WeakAuras.lua

```diff
@@ -878,6 +878,14 @@ local globalConditions =
       state.hastarget = UnitExists("target");
     end
   },
+  ["attackabletarget"] = {
+    display = L["Attackable Target"]
+    type = "bool"
+    events = {"PLAYER_TARGET_CHANGED"},
```

Comment:
There must be a event for when a unit becomes attackable. Though I have no idea where to test that.